cpp/CMakeLists.txt

```diff
@@ -293,7 +296,7 @@ endif(USE_NVTX)
 ###################################################################################################
 # - link libraries --------------------------------------------------------------------------------
 
-target_link_libraries(cuspatial cudf rmm cudart cuda nvrtc)
+target_link_libraries(cuspatial cudf rmm cudart cuda nvrtc gdal)
```

Comment:
Please use the cmake module FindGDAL to ensure goal is found correctly.
https://cmake.org/cmake/help/latest/module/FindGDAL.html

Reply:
Since GDAL (and many others in the future) is installed under a conda env, would it be easier to just add CONDA_LINK_DIRS to link_directories? Some of these third-party libraries may not have FindX CMake module.

Reply:
Done.